kotlin-native/backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/llvm/IrToBitcode.kt

```diff
@@ -2102,6 +2103,22 @@ internal class CodeGeneratorVisitor(val context: Context, val lifetimes: Map<IrE
         }
     }
 
+    private fun storeInlinedParametersDebugInfo(block: IrReturnableBlock) {
+        val inlineFunctionSymbol = block.inlineFunctionSymbol ?: return
+        val function = inlineFunctionSymbol.owner
+        for (parameter in function.valueParameters) {
+            if (parameter.type.isFunctionTypeOrSubtype()) continue
```

Comment:
We don't add any information for inline lambdas since they don't have any address. However we may add some information for noinline limbdas, but would it be of any value during debugging?

Reply:
I think it's fine for now. It probably can be useful if there is a good way to show it, but anyway, you can't see anything meaningful from function reference for now.